src/numeric_index.c

```diff
@@ -640,7 +580,7 @@ void __numericIndex_memUsageCallback(NumericRangeNode *n, void *ctx) {
 
   if (n->range) {
     *sz += sizeof(NumericRange);
-    *sz += n->range->card * sizeof(double);
+    *sz += NR_REG_SIZE; // hll memory size
     if (n->range->entries) {
       *sz += InvertedIndex_MemUsage(n->range->entries);
     }
```

Comment:
Hmm you are correct. Since we track the size of the inverted indexes, and the rest of the range size is constant now, we can calculate it at the tree level